<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperaturas y flujos térmicos por intervalos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnSelFile">
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wScatter1" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="layoutMatplot1"/>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Filtros</string>
      </property>
      <widget class="QPushButton" name="btnNext">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>60</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPrev">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>60</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>59</width>
         <height>59</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSueno">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>984</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Clasificación de sueño</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSedentario">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>984</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Sedentario</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbLigera">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>984</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Actividad ligera</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbModerada">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>984</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Actividad moderada</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
